client-device-connection-infinispan/src/test/resources/application.yaml

```diff
@@ -23,3 +23,4 @@ hono:
       trustStoreType: "PKCS12"
       trustStorePassword: "trust-store-secret"
       useSsl: true
+      sslCiphers: "TLSv1.3 TLSv1.2"
```

Comment:
This example value is at least misleading as its value contains TLS protocol versions but not TLS cipher suite names.